CSS/Styles.css

```diff
@@ -317,48 +339,158 @@ ul {
   height: 20px;
 }
 
-.smallbtn:hover {
-  background: #fff;
-  color: #1a1a1a;
-}
-
-.smallbtn:active {
-  background: #98dede;
-  color: #1a2236;
-}
-
-.projbtn {
-  margin: 1.25em 0 1.25em 1em;
-  padding: 9px 18px 9px 9px;
+.projbtn,
+.projbtn-mod,
+.projbtn-mod-src {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-image: url(../Images/arrow.png);
+  background-color: #1a2236;
+  background-repeat: no-repeat;
+  background-position: center right 12px;
+  background-size: 20px;
+  margin: 1em 0 1.25em 1.2em;
   box-sizing: border-box;
   cursor: pointer;
   height: 50px;
-  background: #1a2236;
+  width: fit-content;
+  padding-left: 2em;
+  padding-right: 5em;
   font-family: 'Poppins', sans-serif;
   font-style: normal;
   font-weight: 400;
   font-size: 16px;
   line-height: 24px;
   text-align: center;
+  vertical-align: middle;
   color: #fff;
   border: 1px solid #fff;
 }
 
-.projbtn img {
-  width: 20px;
-  height: auto;
-  margin-left: 10px;
+.projbtn-mod,
+.projbtn-mod-src {
+  background-image: url(../Images/ic_link.svg);
+  background-color: #fff;
+  color: #1a2236;
+  border: 1px solid #1a2236;
+  width: 15em;
+  margin: 0.75em 2em;
+}
+
+.projbtn-mod-src {
+  background-image: url(../Images/ic_github_white.svg);
 }
 
 .projbtn:hover {
   background: #fff;
   color: #1a2236;
 }
 
+.projbtn-mod:hover,
+.projbtn-mod-src:hover {
+  background-image: url(../Images/ic_github.png);
+  background-color: #1a2236;
+  color: #fff;
+}
+
+.projbtn-mod:hover {
+  background-image: url(../Images/ic_download.png);
+}
+
 .projbtn:active {
   border: 2px solid #1a2236;
 }
 
+.modal-overlay {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  left: -100%;
+  z-index: 99;
+  height: 100%;
+  width: 100%;
+  background-color: #1a2236;
+  opacity: 60%;
+  overflow-y: auto;
+  transition: all 500ms ease-in-out;
+}
+
+.modal-overlay.active {
+  opacity: 100%;
+  left: 0;
+}
+
+.proj-wrp-mod {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  box-sizing: border-box;
+  background-color: #fff;
+  margin: auto;
+  padding: 0;
+  width: 99%;
+  height: 99%;
+}
```

Comment:
- [ ] Please follow the Figma design for the project popup window in the desktop size, as the buttons and the description should have a white background and there is a horizontal scroll in your design please fix that as the responsive design shouldn't have a horizontal scroll.

Your Design | Figma Design
---------------- | -------------------
![Screenshot from 2022-06-16 22-02-16](https://user-images.githubusercontent.com/33752735/174155088-5b7c0381-76a1-4fa0-a7c3-5365b8b0256e.png) | ![Screenshot from 2022-06-16 22-01-04](https://user-images.githubusercontent.com/33752735/174155127-c9e56e7d-9ab0-4997-8fe2-f413c1c2a06f.png)

Reply:
The figma template for the modal requires scroll on Y axis. It is even a taller aspect ratio, so it's not possible to make it responsive and wider as in any computer display without scrolling. I did my best trying to adapt the design in a pleasant way. The buttons out of the container were a mistake I'm fixing now.
Thanks.

Reply:
By horizontal I mean X axis.

Reply:
__You are totally right. I'm sorry. This project has been very taxing on me.__